INSTALL.md

```diff
@@ -168,6 +168,21 @@ This assumes you already installed the cross-built `libelf` on the sysroot
 directory.  Otherwise, you can also build it from source (please see below) or
 use it on a different path using `--with-elfutils=<PATH>`.
 
+To compile for Android 9+, export the CC environment variable and disable the not
+yet implemented python and libstdc++ support. E.g. to configure for Android
+AArch64 do
+
+    $ export CC=$NDK/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android33-clang
+    $ ./configure --arch=aarch64 --cross-compile=aarch64-linux-gnu- --without-libpython --without-libstdc++
```

Comment:
I see an error as follows.
```
$ make
  CC       uftrace.o
  CC       arch/aarch64/cpuinfo.o
  LINK     arch/aarch64/uftrace.o
/bin/sh: 1: aarch64-linux-gnu-ld: not found
make[1]: *** [Makefile:22: /home/honggyu/work/uftrace/arch/aarch64/uftrace.o] Error 127
make: *** [Makefile:293: /home/honggyu/work/uftrace/arch/aarch64/uftrace.o] Error 2
```
Adding one more line for `LD` makes it successful.
```
$ export CC=$NDK/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android33-clang
$ export LD=$NDK/toolchains/llvm/prebuilt/linux-x86_64/bin/ld.lld
$ ./configure --arch=aarch64 --cross-compile=aarch64-linux-gnu- --without-libpython --without-libstdc++
```

Reply:
That's interesting, I can't repro. BTW isn't `LD=$CC` enough in your case?

Reply:
With the current instruction guide, `.config.` file becomes as follows.
```
$ cat .config 
    ...
override ARCH   := aarch64
override CC     := /home/honggyu/toolchains/android-ndk-r25b/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android33-clang
override LD     := aarch64-linux-gnu-ld
    ...
```
But there is no `aarch64-linux-gnu-ld` file in android NDK. It might be the case that your system already has `aarch64-linux-gnu-ld` somewhere else outside of android NDK. I think we also need
```
export LD=$NDK/toolchains/llvm/prebuilt/linux-x86_64/bin/ld.lld
```
along with `export CC`.

Reply:
This problem is shown in @kangtegong's environment and having `export LD=.../ld.lld` fixed his problem as well.

Reply:
Makes sense, I had AArch64 binutils installed so couldn't repro. I updated INSTALL.md.